src/main/kotlin/no/nav/familie/ks/sak/kjerne/tilbakekreving/TilbakekrevingService.kt

```diff
@@ -31,4 +47,35 @@ class TilbakekrevingService(
         eksisterendeTilbakekreving?.let { tilbakekrevingRepository.deleteById(it.id) }
         return tilbakekrevingRepository.save(tilbakekreving)
     }
+
+    fun hentForhåndsvisningTilbakekrevingVarselBrev(
+        behandlingId: Long,
+        forhåndsvisTilbakekrevingsvarselbrevDto: ForhåndsvisTilbakekrevingsvarselbrevDto
+    ): ByteArray {
+        val vedtak = vedtakRepository.findByBehandlingAndAktivOptional(behandlingId) ?: throw Feil(
+            "Fant ikke vedtak for behandling $behandlingId ved forhåndsvisning av varselbrev for tilbakekreving."
+        )
+        val personopplysningGrunnlag = personopplysningGrunnlagService.hentAktivPersonopplysningGrunnlagThrows(behandlingId)
+        val søker = personopplysningGrunnlag.søker
+        val arbeidsfordeling = arbeidsfordelingService.hentArbeidsfordelingPåBehandling(behandlingId)
+        return tilbakekrevingKlient.hentForhåndsvisningTilbakekrevingVarselbrev(
+            forhåndsvisVarselbrevRequest = ForhåndsvisVarselbrevRequest(
+                varseltekst = forhåndsvisTilbakekrevingsvarselbrevDto.fritekst,
+                ytelsestype = Ytelsestype.KONTANTSTØTTE,
+                behandlendeEnhetId = arbeidsfordeling.behandlendeEnhetId,
+                behandlendeEnhetsNavn = arbeidsfordeling.behandlendeEnhetNavn,
+                språkkode = søker.målform.tilSpråkkode(),
+                feilutbetaltePerioderDto = FeilutbetaltePerioderDto(
+                    sumFeilutbetaling = simuleringService.hentFeilutbetaling(behandlingId).longValueExact(),
+                    perioder = hentTilbakekrevingsperioderISimulering(simuleringService.hentSimuleringPåBehandling(behandlingId))
+                ),
+                fagsystem = Fagsystem.KONT, // for ks vil fagsystem alltid være KONT,
+                eksternFagsakId = vedtak.behandling.fagsak.id.toString(),
+                ident = søker.aktør.aktivFødselsnummer(),
+                saksbehandlerIdent = SikkerhetContext.hentSaksbehandlerNavn(),
+                verge = null, // TODO kommer når verge er implementert
```

Comment:
Skal vi ha verge for kontantstøtte i det hele tatt?
Lurer egentlig på om det er det samme som institusjon, at vi ikke skal ha det.
Derfor kan TODO fjernes, eller?

Reply:
Trenger du forresten å sette verge og institusjon til null her?
De defaulter uansett til null, og skal som jeg vet ikke brukes for KS.

Reply:
@UyQuangNguyen Kanskje jeg er feil, men så lenge jeg kan huske, det er mulig å ha verge for KS også. Kan spørre @EAamli etter ferien